tests/test_cli.py

```diff
@@ -0,0 +1,146 @@
+"""
+Tests cosmicqc cli module
+"""
+
+import pathlib
+
+from pyarrow import parquet
+
+from .utils import run_cli_command
+
+
+def test_cli_util():
+    """
+    Test the `identify_outliers` function of the CLI.
+    """
+
+    command = """echo 'hello world'"""
+    stdout, stderr, returncode = run_cli_command(command)
+
+    assert returncode == 0
+
+
+def test_cli_identify_outliers(tmp_path: pathlib.Path, basic_outlier_csv: str):
+    """
+    Test the `identify_outliers` function of the CLI.
+    """
+
+    stdout, stderr, returncode = run_cli_command(
+        (
+            f"""cosmicqc identify_outliers --df {basic_outlier_csv}"""
+            """ --feature_thresholds {"example_feature":1.0}"""
+            f" --export_path {tmp_path}/identify_outliers_output.parquet"
+        )
+    )
```

Comment:
The `basic_outlier_csv` is a string of a path correct?

Reply:
Thanks for checking, yes this is a string filepath compatible with the command line.